pom.xml

```diff
@@ -734,17 +731,6 @@
             </properties>
         </profile>
 
-        <profile>
-            <id>jdk-18-plus</id>
-            <activation>
-                <jdk>[18,)</jdk>
-            </activation>
-            <properties>
-                <!-- snippet-path only works with JDK 18+ -->
-                <additionalJavadocOption>--snippet-path=${basedir}/src/test/java:${basedir}/service-custom/s3-transfer-manager/src/test/java</additionalJavadocOption>
```

Comment:
Had to remove this because I can't get it working with maven javadoc aggregation plugin :(